lib/image_composition.dart

```diff
@@ -0,0 +1,167 @@
+import 'dart:async';
+import 'dart:ui';
+
+import 'extensions.dart';
+
+export 'extensions.dart';
+
+class _Composed {
+  /// The image that will be composed.
+  final Image image;
+
+  /// The position where the [image] will be composed.
+  final Vector2 position;
+
+  /// The source on the [image] that will be composed.
+  final Rect source;
+
+  final double rotation;
```

Comment:
/// The angle (in radians) used to rotate the image around the [anchor]

Reply:
The variable should maybe be called angle too, since that is what it is called in the Effects and PositionComponent?

Reply:
Agreed, updated it where needed.